nix/hetzner.nix

```diff
@@ -90,12 +90,177 @@ with lib;
         part swap --recommended --label=swap --fstype=swap --ondisk=vda
         part / --fstype=ext4 --label=root --grow --ondisk=vda
       '';
-      type = types.lines;
+      type = types.nullOr types.lines;
       description = ''
         Specify layout of partitions and file systems using Anacondas Kickstart
         format. For possible options and commands, please have a look at:
 
         <link xlink:href="http://fedoraproject.org/wiki/Anaconda/Kickstart"/>
+
+        If the Kickstart is not sufficient for your partitioning needs,
```

Comment:
Probably remove the `the`: `If Kickstart is not sufficient...`

Reply:
Done